python/Makefile

```diff
@@ -1,4 +1,39 @@
-### Testing ###
+### project constants ###
+ZERO2AIHOME ?= ${HOME}/sandbox/zero-to-ai
+
+.FORCE:
+
+### python versioning ###
+export PYTHONPATH=$(ZERO2AIHOME)/python
```

Comment:
can you remove this line? we probably don't need to set the `PYTHONPATH` variable. verify this from `make test-assign0` as well as local execution of Karel programs (i.e. `python hailstorm_karel.py`)